contrib/main/plugins.yml

```diff
@@ -24,3 +24,11 @@
   description: |
     Tutor plugin to install Meilisearch and enable course content search in Studio.
     See the README on GitHub for further setup instructions.
+
+- name: wordpress
+  src: tutor-contrib-wordpress>=1.1.0,<2.0.0
```

Comment:
I think you want to unpin the version here. People will always want to install the latest release in the main branch, right?

Reply:
You're absolutely right. Should I follow the editable installation for main branch too?

Reply:
It's up to you!

Reply:
Done